snapcraft.yaml

```diff
@@ -23,45 +23,31 @@ apps:
     command: certbot
     environment:
       PATH: "$SNAP/bin:$SNAP/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games"
-      AUGEAS_LENS_LIB: $SNAP/usr/share/augeas/lenses/dist
+      AUGEAS_LENS_LIB: "$SNAP/usr/share/augeas/lenses/dist"
+      LD_LIBRARY_PATH: "$SNAP/usr/lib/x86_64-linux-gnu/:$LD_LIBRARY_PATH"
   renew:
     command: certbot -q renew
     daemon: oneshot
     environment:
       PATH: "$SNAP/bin:$SNAP/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games"
       AUGEAS_LENS_LIB: $SNAP/usr/share/augeas/lenses/dist
+      LD_LIBRARY_PATH: "$SNAP/usr/lib/x86_64-linux-gnu/:$LD_LIBRARY_PATH"
     passthrough:
         # Run approximately twice a day with randomization
         timer: 00:00~24:00/2
 
 parts:
-  python-augeas:
-    plugin: python
-    source: git://github.com/basak/python-augeas
-    source-branch: snap
-    python-version: python2
-    build-packages: [libaugeas0]
-  certbot:
-    plugin: python
-    source: git://github.com/certbot/certbot
-    source-branch: master
-    python-version: python2
-    after: [python-augeas]
-  certbot-nginx:
+  acme:
     plugin: python
     source: git://github.com/certbot/certbot
-    source-branch: master
-    source-subdir: certbot-nginx
-    build-packages: [libaugeas-dev]
-    stage-packages: [libaugeas0]
-    after: [certbot]
-    python-version: python2
-  certbot-apache:
+    source-branch: v0.35.1
```

Comment:
Why is this now tied to a particular tag rather than following master? Won't this stop edge updating automatically for us?

Reply:
Indeed, it is a work in progress, I need to find a way to dynamically configure the build. It is related to the other conversations.